src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
```

Comment:
The background-image property uses a relative path to the 'star.svg' file. Ensure that this path is correct and that the file exists at the specified location to avoid broken image links.